File: SDKNodeJs/models/MajCompteModel.js
module.exports = MajCompteModel;

function MajCompteModel(obj) {
    this.CodeUtilisateur = obj.CodeUtilisateur;
    this.TypeAbonnement = obj.TypeAbonnement;
    this.DateSouscription = obj.DateSouscription;
    this.DateExpiration = obj.DateExpiration;
    this.Tarif = obj.Tarif;
    this.Statut = obj.Statut;
}

File: python_django/sdk_lpl/models/MajCompteModel.py
class MajCompteModel(object):
    def __init__(self, code_utilisateur, type_abo, date_souscription, date_expiration, tarif, statut):
        self.CodeUtilisateur = code_utilisateur
        self.TypeAbonnement = type_abo
        self.DateSouscription = date_souscription
        self.DateExpiration = date_expiration
        self.Tarif = tarif
        self.Statut = statut
